### PR TITLE
Dat 3215

### PR DESCRIPTION
**Pull request by rybmat**
healthcheck for workers that checks if worker is processing when queue is not empty

### CODE REVIEW

File: workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerApplication.java [hunk 1]
```diff
@@ -1,12 +1,14 @@
 package com.wikia.workers.articlesnippetworker;
 
+import com.wikia.workers.articlesnippetworker.configuration.ArticleSnippetWorkerConfiguration;
+import com.wikia.workers.articlesnippetworker.guice.ArticleSnippetWorkerModule;
 import com.wikia.workers.common.WorkerApplication;
 import com.wikia.workers.common.WorkerConfiguration;
```
Comment (WikiaTech) [inline workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerApplication.java]: MINOR Remove this unused import 'com.wikia.workers.common.WorkerConfiguration'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

File: workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerApplication.java [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 import com.wikia.workers.common.guice.WorkerModule;
```
Comment (WikiaTech) [inline workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerApplication.java]: MINOR Remove this unused import 'com.wikia.workers.common.guice.WorkerModule'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

File: workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerCommand.java [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package com.wikia.workers.articlesnippetworker;
 
-import com.wikia.workers.common.AbstractWorker;
+import com.wikia.workers.articlesnippetworker.configuration.ArticleSnippetWorkerConfiguration;
 import com.wikia.workers.common.WorkerCommand;
 import com.wikia.workers.common.WorkerConfiguration;
```
Comment (WikiaTech) [inline workers/article-snippet-worker/src/main/java/com/wikia/workers/articlesnippetworker/ArticleSnippetWorkerCommand.java]: MINOR Remove this unused import 'com.wikia.workers.common.WorkerConfiguration'. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MINOR 4 minor

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MINOR [Remove this unused import 'com.wikia.workers.common.AbstractWorker'.](https://github.com/Wikia/pandora/blob/3f5a1b13e5c8d95d8c9cb006e9331fc526827ffa/workers/delete-worker/src/main/java/com/wikia/workers/deleteworker/DeleteWorkerCommand.java#L3) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by SebastianMarzjan** [comment]
While I keep wondering about some simpler healthcheck, this should handle all of the typical problems that we have.

**Pull request merged**